numpyro/infer/mcmc.py

```diff
@@ -382,11 +383,76 @@ def _compile(self, rng_key, *args, extra_fields=(), init_params=None, **kwargs):
         except TypeError:
             pass
 
+    @property
+    def init_state(self):
+        """
+        The initial state to run MCMC. If this attribute is not None,
```

Comment:
nit:
The initial state of the MCMC chain. If this attribute is `None`, 
:meth:`run` will call the `self.sampler.init(...)` method for initialization.